### PR TITLE
workspace module

### PR DESCRIPTION
**Pull request by aviatesk**
and bunch of improvements:
- show variables correctly (we need `all` option for modules other than `Main`)
- simplify TS code
- update workspace on module selection, REPL init

remaining TODOs:
- [x] add icon for workspace view to toggle workspace module selector
- [ ] fix `show` method -- it currently fails on `show`ing `Enum` and thus we can't get workspace for `Base`, for instance

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -618,16 +622,16 @@
         "viewsContainers": {
             "activitybar": [
                 {
-                    "id": "julia-explorer",
-                    "title": "Julia Explorer",
+                    "id": "julia-workspace",
+                    "title": "Julia Workspace",
                     "icon": "images/julia-dots.svg"
```
Comment (aviatesk) [inline package.json:627]: I tried to unify workspace/REPL variable/explorer terminology, but let me know if you don't like this.

**Comment by pfitzseb** [comment]
Kinda, but that doesn't work for packages (because they are not necessarily reachable from `Main`).

**Comment by davidanthoff** [comment]
If you load a package in the REPL, you can access it from Main, right?

**Comment by aviatesk** [comment]
close in favor of #1302 . Small enhancements in this PR will be added afterwards.

**Pull request closed without merging**